opentracing-api-extensions-tracer/src/test/java/io/opentracing/contrib/api/tracer/APIExtensionsFullSpanLifecycleTest.java

```diff
@@ -51,19 +38,6 @@ public void testMockTracerManualSpan() {
         Mockito.verify(spanObserver, Mockito.times(1)).onFinish(Matchers.any(SpanData.class), Matchers.anyLong());
     }
 
-    @Test
-    public void testNoopTracerActiveSpan() {
-        APIExtensionsTracer extTracer = new APIExtensionsTracer(NoopTracerFactory.create());
-        TracerObserver tracerObserver = Mockito.mock(TracerObserver.class);
-        SpanObserver spanObserver = Mockito.mock(SpanObserver.class);
-        Mockito.when(tracerObserver.onStart(Matchers.any(SpanData.class))).thenReturn(spanObserver);
-        extTracer.addTracerObserver(tracerObserver);
-
-        extTracer.buildSpan("testOp").startActive(true).close();
-        
-        Mockito.verify(spanObserver, Mockito.times(1)).onFinish(Matchers.any(SpanData.class), Matchers.anyLong());
-    }
-
     @Test
     public void testNoopTracerManualSpan() {
```

Comment:
nit: as there is no distinction now between manual and active, could you change this test name to `testNoopTracerSpan`?